examples/h2_server/source/app.d

```diff
@@ -0,0 +1,27 @@
+import vibe.appmain;
+import vibe.http.server;
+import vibe.stream.tls;
+import vibe.stream.botan;
+import vibe.core.log;
+import vibe.core.core;
+import libasync.threads;
+import std.datetime;
+
+void handleRequest(scope HTTPServerRequest req, scope HTTPServerResponse res)
+{
+	res.writeBody(cast(ubyte[])"Hello, World!", "text/plain");
```

Comment:
Definitely right about the missing `const` (PR is merged). But there is also a `string` overload of `writeBody`, so that the cast still shouldn't be necessary.